modules/@apostrophecms/login/index.js

```diff
@@ -196,6 +197,22 @@ module.exports = {
             return user;
           }
         } : {})
+      },
+      get: {
+        async context () {
+          let aposPackage = {};
+          try {
+            aposPackage = await fs.readJson('./node_modules/apostrophe/package.json');
```

Comment:
@boutell Stuart wanted the Apostrophe version, not the project one. Is this correct or should I use self.apos.rootDir?

Reply:
This code is inside the apostrophe module, therefore you can use relative paths within the module. Don't try to read from node_modules.

Reply:
Use require instead. require can read json files, and it is always relative to the file making the require call.